libs/remix-ui/debugger-ui/src/lib/vm-debugger/global-variables.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react' // eslint-disable-line
+import DropdownPanel from './dropdown-panel' // eslint-disable-line
+import { BN } from 'ethereumjs-util'
+
+export const GlobalVariables = ({ block, receipt, tx }) => {
+  console.log(block, receipt, tx)
+  // see https://docs.soliditylang.org/en/latest/units-and-global-variables.html#block-and-transaction-properties
+  const globals = {
+    'block.basefee': (new BN(block.baseFeePerGas.replace('0x', ''), 'hex')).toString(10) + ` Wei (${block.baseFeePerGas})`,
+    'block.chainid': tx.chainId,
+    'block.coinbase': block.miner,
+    'block.difficulty': block.difficulty,
+    'block.gaslimit': block.gasLimit,
+    'block.number': block.number,
+    'block.timestamp': block.timestamp,
+    'msg.sender': tx.from,
+    'msg.sig': tx.input.substring(0, 10),
+    'msg.value': tx.value + ' Wei',
+    'tx.origin': tx.from
+  }
+  return (
+    <div id='globalvariable' data-id='globalvariable'>
+      <DropdownPanel hexHighlight={false} bodyStyle={{ fontFamily: 'monospace' }} dropdownName='Global Variables' calldata={globals || {}} />
```

Comment:
<img width="284" alt="Screenshot 2021-08-17 at 12 19 17 PM" src="https://user-images.githubusercontent.com/30843294/129682968-35af2248-7676-4068-b16d-d068b91cfa75.png">

I think title at LHS should be in one line for better UI